client/src/api.ts

```diff
@@ -25,35 +46,144 @@ export const createElection = async (data: {
-    }
+    return { mailElections: [mailElection] }
   }
-  return await fetchJSON('/elections')
+  return await fetchJSON(`/api/mailelection`)
 }
 
 export const getElection = async (
   electionId: string
-): Promise<ElectionsListItem> => {
+): Promise<MailElection> => {
   if (useFakeAPI) {
-    return {
-      id: electionId,
-      name: 'foo',
-      createdAt: new Date().toISOString(),
-    }
+    return mailElection
   }
-  return fetchJSON(`/elections/${electionId}`)
+  return fetchJSON(`/api/mailelection/${electionId}`)
 }
+
+export const approveElection = async (data: {
+  electionId: string
+  userId: string
+}) =>
+  await fetchJSON(`/api/mailelection/${data.electionId}/approve`, {
+    method: 'post',
+    body: JSON.stringify(data),
+  })
+
+export const putElectionDefinition = async ({
+  electionId,
+  election,
+}: {
+  electionId: string
+  election: Election
+}) =>
+  await fetchJSON(`/api/mailelection/${electionId}/definition`, {
+    method: 'put',
+    body: JSON.stringify(election),
+  })
+
+export const getElectionDefinition = async ({
+  electionId,
+}: {
+  electionId: string
+}): Promise<Election> =>
+  await fetchJSON(`/api/mailelection/${electionId}/definition`)
+
+export const putBallotTemplate = async ({
+  ballotStyleId,
+  electionId,
+  file,
+  precinctId,
+}: {
+  ballotStyleId: string
+  electionId: string
+  file: Buffer
+  precinctId: string
+}) =>
+  await fetchJSON(
+    `/api/mailelection/${electionId}/ballot-style/${ballotStyleId}/precinct/${precinctId}/template`,
+    {
+      method: 'put',
+      body: file,
+    }
+  )
+
+export const getBallotTemplate = async ({
+  ballotStyleId,
+  electionId,
+  precinctId,
+}: {
+  ballotStyleId: string
+  electionId: string
+  precinctId: string
+}): Promise<BallotTemplate> =>
+  await fetchJSON(
+    `/api/mailelection/${electionId}/ballot-style/${ballotStyleId}/precinct/${precinctId}/template`
+  )
+
+export const putMailingList = async ({
+  electionId,
+  voterMailingListFile,
+}: {
+  electionId: string
+  voterMailingListFile: Buffer
+}) =>
+  await fetchJSON(`/api/mailelection/${electionId}/mailinglist`, {
+    method: 'put',
+    body: voterMailingListFile,
+  })
+
+export const getVoters = async ({
+  electionId,
+}: {
+  electionId: string
+}): Promise<{ voters: Voter[] }> =>
+  await fetchJSON(`/api/mailelection/${electionId}/voters`)
+
+export const getInserts = async ({
+  electionId,
+}: {
+  electionId: string
+}): Promise<InsertsData> =>
+  await fetchJSON(`/api/mailelection/${electionId}/inserts`)
+
+export const putInserts = async ({
+  electionId,
+  insertsData,
+}: {
+  electionId: string
+  insertsData: InsertsData
+}) =>
+  await fetchJSON(`/api/mailelection/${electionId}/inserts`, {
+    method: 'put',
+    body: JSON.stringify(insertsData),
+  })
+
+export const sendBatch = async ({
+  electionId,
+  batchName,
+}: {
+  electionId: string
+  batchName: string
+}) =>
+  await fetchJSON(`/api/mailelection/${electionId}/batches/send`, {
```

Comment:
see my comments on `API.md` for what this should be. Do we have a UX for this so we can really make sure we're building the right API?